### PR TITLE
Remodelling workflow (test_rbdV2) and sdk layer for rbd

### PR DESCRIPTION
**Pull request by VasishtaShastry**
# Description

- Moved sdk layer from multiple inheritance to composition
- Remodelled  workflow layer for composition
- Added utility block to workflow module
- Added option to select clusters at test_config level
- Added demo test_configs and suits

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by VasishtaShastry** [comment]
This PR is a followup PR of https://github.com/red-hat-storage/cephci/pull/1359/

It was discussed to continue and fix the workflow layer according to remodelled sdk layer
We request To review test_rbdV2 (workflow layer)
The demo was tested with test_config - tests/rbd/test_configs/feature_enable_cli_test.yaml
Suite - suites/pacific/rbd/demo_suite_rbd_workflow_usage.yaml

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-R6748L/rbd_featue_enable_0.log

Will squash commits once its reviewed, which would solve the issue of DCO.

**Comment by VasishtaShastry** [comment]
Have got request to convert the current model from expecting complete command from user to two words
service and action, working on it

File: tests/rbd/test_configs/feature_enable_cli_test.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: rbd-feature-enable-test
+description: 'create pool and enable features'
+steps:
+  - command: ceph osd pool create
```
Comment (HaruChebrolu) [inline tests/rbd/test_configs/feature_enable_cli_test.yaml:5]: I don't see the purpose of having seperate modules for commands if we specify command explicitly in test-config.
@psathyan what are your thoughts?

Comment (psathyan) [reply tests/rbd/test_configs/feature_enable_cli_test.yaml:5]: @VasishtaShastry I agree with @HaruChebrolu review that this conflicts with the upcoming changes.

**Comment by VasishtaShastry** [comment]
Closing the PR based on the conversation on May 19th.
RBD FG will follow existing method of writing modules without SDK and (workflow layer - configs for TCs) until it wil be implemented at team level.

**Pull request closed without merging**